omero/institution/getting-started.html

```diff
@@ -11,6 +11,78 @@
             <h3 class="header-green">Getting Started</h3>
             <p class="subheader">Whether you’re an experienced system administrator ready to jump straight into installing OMERO.server or a manager wondering if the OMERO platform will meet your facility’s needs, the following guides will get you started.</p>
         </div>
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h3>What is needed to host OMERO?</h3>
+        </div>
+        <hr>
+        <div class="row">
+            <div class="medium">
+                <p>The resources needed for setting up, keeping and maintaining OMERO vary according to your image data sizes and quality, and the infrastructure in your institution. Nevertheless, some general hints are given here as points which you should think about and implement in your concrete case.</p>
+                <p>You can benefit from asking your colleagues at Institutions which are already running OMERO. For that, we also list some contacts to regional groups and list of selected institutions which run OMERO below.</p>
+            </div>
+            <div>
+                <h5>Server hardware and environment</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html" target="_blank">Hardware</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html" target="_blank">Storage</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html" target="_blank">RAM</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html" target="_blank">CPU</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html" target="_blank">Software dependencies</a></li>
```

Comment:
These could have additional anchors a la https://docs.openmicroscopy.org/omero/5.6.3/sysadmins/system-requirements.html#software

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/f62d027dd1f89d4307748f0a7236d1b84f072d18